benchexec/tools/metaval++.py

```diff
@@ -74,15 +84,8 @@ def get_value_from_output(self, output, identifier):
         # only the first group will be returned.
         match = None
         for line in output:
-            matches = re.findall(identifier, line)
-            if matches is not None and len(matches) > 0:
-                if isinstance(matches[0], tuple):
-                    logging.warning(
-                        "The regex '%s' has groups, but only the first group will be returned",
-                        identifier,
-                    )
-                    match = matches[0][0]
-                else:
-                    match = matches[0]
+            matches = re.search(identifier, line).group()
```

Comment:
This will crash because `search()` returns `None` if it does not find anything.

Reply:
Thanks a lot for pointing this out!

Has been fixed